routes/api.php

```diff
@@ -33,6 +33,9 @@
     }
     $item->code = $code->code;
     $item->offset_top = $code->offset_top;
+    if (!empty($code->author_name)) {
+        $item->author = $code->author_name;
```

Comment:
This attribute was renamed to `authors` in the new API.
```suggestion
        $item->authors = $code->author_name;
```